### ISSUE TITLE
Block Breaks and interactions are not visualized

### ISSUE THREAD

**Issue by ProgrammerDan** [Scuwr/Snitch-Visualizer#9]
Currently block break and interaction information is not displayed as an overlay. This would be very nice to have.

**Comment by staygroovy57** [comment]
It would be as long as the user has a way to turn them off (expiring the display) after visualizing or whenever they want? Otherwise, how long would it stay visible?

**Comment by ProgrammerDan** [comment]
My thought is to add a new key you could bind, to clear on command. Otherwise they will persist, to allow you to get whatever "angle" on the recorded activities desired. As you investigate using /jainfo, new block breaks/access/etc will appear, until you hit the "clear" key.

**Comment by PromoFaux** [comment]
There was a version of this mod on 1.7.10 that visualised blockbreaks and interactions, and it had a really annoying way of clearing it. I can't remember exactly, but it was something along the lines of Ctrl + Alt, and then restarting the minecraft client. A simple toggle in the GUI would be lovely.

**Comment by ProgrammerDan** [comment]
Yeah, it was the original snitch visualizer, and it was more like 1.6.4 :D

I agree whole-heartedly! I've been pulled off on Civcraft main mods for a while, but I'll get back to this if Scuwr doesn't beat me to it.

**Comment by PromoFaux** [comment]
Ah right! I liked the feature, just hated the way of clearing it down.

I might even take a look myself. One day. Must have another go at getting your image up and running.
